test/e2e_node/e2e_node_suite_test.go

```diff
@@ -205,8 +208,13 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// Reference common test to make the import valid.
 	commontest.CurrentSuite = commontest.NodeE2E
 
-	return nil
-}, func([]byte) {
+	// ginkgo would spawn multiple processes to run tests.
+	// Since the bearer token is generated randomly at run time,
+	// we need to distribute the bearer token to other processes to make them use the same token.
+	klog.Infof("Distribute bearer token: %s", framework.TestContext.BearerToken)
+	return []byte(framework.TestContext.BearerToken)
+}, func(token []byte) {
+	framework.TestContext.BearerToken = string(token)
```

Comment:
this seems fine as per the docs for SynchronizedBeforeSuite
https://github.com/onsi/ginkgo/blob/master/ginkgo_dsl.go#L504

just to double check my understanding here, does this mean that the rest of framework.TestContext is empty when the child processes are run and only the token field is set?

Reply:
> does this mean that the rest of framework.TestContext is empty when the child processes are run and only the token field is set?

nope, `framework.TestContext` is set like the parent process and the token field is another random token.

Reply:
ok, i see. so this persists the original token that we generated in the parent process.

Reply:
yeah you are right.